docs/code_conventions/README.MD

```diff
@@ -34,6 +34,41 @@ apiVersion = "v1"
 username = demisto.params().get("url")
 ```
 
+## Arguments
+When naming an argument, the name should be all lowercase with spaces replaced by underscores. See the example below:
+
+### Do this:
+```
+email_header
```

Comment:
give full command example 
`!cylance-get-incident incident_id` instead of `!cylance-get-incident incidentId`